main.qml

```diff
@@ -1465,20 +1427,22 @@ ApplicationWindow {
         }
     }
 
-    // Confrirmation aka question dialog
-    StandardDialog {
+    // Transaction confirmation popup
+    TxConfirmationDialog {
+        // dynamically change onclose handler
+        id: txConfirmationPopup
         z: parent.z + 1
-        id: transactionConfirmationPopup
```

Comment:
?

Reply:
not sure what you mean here

Reply:
Why did you rename it?

Reply:
to be consistent with saveTxDialog and successfulTxPopup